### DISCUSSION TITLE
Is it possible to add my own experimental finetuned models locally?

### DISCUSSION THREAD

**Discussion by nofreewill42** [open-webui/open-webui#719] [General]
I would like to create some training data by hand and do a bunch of finetunings to observe how the model's behaviour is changing.
Is this possible to do right now?

**Comment by tjbck** [comment]
Could you elaborate more on what you're trying to achieve? For now you would have to fine-tune the model externally, and you should be able to import the fine-tuned model to Ollama as a GGUF file.

**Reply by nofreewill42** [reply]
You are correct, thank you very much!
